cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -9,6 +9,12 @@ metadata:
   name: rook-ceph-cluster
   namespace: rook-ceph
 ---
+apiVersion: v1
```

Comment:
Declaring it in the cluster.yaml is the expected pattern, rather than having the operator to create the roles as was the pattern before the change to service accounts a couple months ago. No need to change this as postulated in the original PR comments